libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/resources/testProject/kt-63230/TEST-wasm.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<results>
+  <testsuite name="PrintTest" tests="1" skipped="0" failures="0" errors="0" timestamp="..." hostname="..." time="...">
+    <properties />
+    <testcase name="test[wasmJs, browser, ChromeHeadless122.0.6261.95, Windows10]" classname="PrintTest" time="..." />
```

Comment:
Is there any way to make the browser information disappear here?

Reply:
@ilgonmic ^^

Reply:
It seems like it has been handled already, but we only checked for `[browser` in the name of the test. I changed it so that the current name is also rewritten properly.